shopgate/pwa
# Automated changelog creation

# Description

Starting from now, the changelogs will now be created upon version builds using `lerna-changelog`.

## Type of change

Please add an "x" into the option that is relevant:


## How to test it

Check the changelog after creating a release. Make sure, that it is not duplicating any entries, when the "latest" version title is already placed in the file.
